Project.toml

```diff
@@ -34,7 +34,7 @@ JSON3 = "1"
 LazyArtifacts = "1"
 NLopt = "0.5, 0.6"
 PooledArrays = "0.5, 1"
-ProgressMeter = "1.5"
+ProgressMeter = "1.5, 1.6, 1.7"
```

Comment:
for versions > 1.0, specifying a minor version just gives a lower bound

(the problem is that we have a lot of pre-1.0 dependencies where minor versions are considered breaking)

Reply:
So this change will have no effect in practice?  If so, I will back it out.

Reply:
correct

Reply:
I will take a look to see when ProgressUnknown and other features that we now use were introduced and set it accordingly.

Reply:
i backed out the change